babbage/query/cuts.py

```diff
@@ -18,14 +19,28 @@ def cut(self, ast):
             raise QueryException('Invalid cut: %r' % ast[0])
         self.results.append((ast[0], ast[1], value))
 
+    def _check_type(self, ref, value):
+        model_type = self.cube.model[ref].datatype
+        if type(value) is str and model_type == 'string':
```

Comment:
To keep babbage clean, I think this string check should be done using six: see https://pythonhosted.org/six/#six.string_types